docs/synapse/userguides/storm_ref_automation.ipynb

```diff
@@ -11,30 +43,675 @@
     "Storm Reference - Automation\n",
     "============================\n",
     "\n",
+    ".. _auto-bkgd:\n",
+    "\n",
+    "Background\n",
+    "----------\n",
+    "\n",
+    "Synapse is designed to facilitate large-scale analysis over disparate data sources with speed and efficiency. Many of the features that support this analysis are built into Synapse’s architecture, from performance-optimized indexing and storage to an extensible data model that allows you to reason over data in a structured manner.\n",
+    "\n",
+    "Synapse also supports large-scale analysis through the use of **automation.** Synapse’s automation features are available through the Storm runtime and include:\n",
+    "\n",
+    "- `Triggers and Cron`_\n",
+    "- `Macros`_\n",
```

Comment:
Storm Dmons need to be listed here, even if we leave a sectino for them marked as TDB